planner/core/testdata/integration_serial_suite_in.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "name": "TestReadFromStorageHint",
+    "cases": [
+      "desc select avg(a) from t",
+      "desc select /*+ read_from_storage(tiflash[t]) */ avg(a) from t",
+      "desc select /*+ read_from_storage(tiflash[t]) */ sum(a) from t",
+      "desc select /*+ read_from_storage(tiflash[t]) */ sum(a+1) from t",
+      "desc select /*+ read_from_storage(tiflash[t]) */ sum(isnull(a)) from t",
+      "desc select /*+ READ_FROM_STORAGE(TIKV[t1], TIKV[t2]) */ * from t t1, t t2 where t1.a = t2.a",
```

Comment:
Also add a test case for hint with both storages? e.g. `select /*+ read_from_storage(tikv[t1], tiflash[t2]) */ * from t t1, t t2 where t1.a = t2.a`

Reply:
It's correct.

Reply:
done